### ISSUE TITLE
spago upgrade-set from 0.13.8 to 0.14.4

### ISSUE THREAD

**Issue by FranklinChen** [exercism/purescript#220]
Currently https://github.com/purescript/package-sets/releases/download/psc-0.13.8-20210226/packages.dhall is being used. It's probably time to update to 0.14.x, e.g. https://github.com/purescript/package-sets/releases/download/psc-0.14.4-20210919/packages.dhall as of now.

**Comment by emiel** [comment]
Thanks for the feedback @FranklinChen. This is indeed on the wishlist. One thing that is a blocker at the moment is that the latest PureScript emits a lot of new warnings. I was kind of waiting for the ecosystem to catch up and fix a lot of these so as not to confront those new to PureScript with these. I'll have a look around and see how far we are.

**Comment by MatthijsBlom** [comment]
Wat could a passer-by do to help with this?

**Comment by FranklinChen** [comment]
Meanwhile, a year later, PureScript appears to be at version 0.15.4. https://github.com/purescript/purescript/releases/tag/v0.15.4

I admit I haven't been doing anything with PureScript for a while because it moves so fast.

**Comment by FranklinChen** [comment]
Also, PureScript has been at 0.15.15 https://github.com/purescript/purescript/releases/tag/v0.15.15 since February 2024, a year ago. Are volunteers needed to keep the PureScript track up to date?

**Comment by IsaacG** [comment]
> Are volunteers needed to keep the PureScript track up to date?


Yes! If you're interested in helping maintain, please open a thread on the forum.

**Comment by FranklinChen** [comment]
I'm currently waiting for my test runner update https://github.com/exercism/purescript-test-runner/pull/56 to be merged in...
